dvc/cli/__init__.py

```diff
@@ -165,9 +165,9 @@ def main(argv=None):  # noqa: C901, PLR0912, PLR0915
     # the copied parent's standard file descriptors. If we make any logging
     # calls in this state it will cause an exception due to writing to a closed
     # file descriptor.
-    if sys.stderr.closed:  # pylint: disable=using-constant-test
+    if not sys.stderr or sys.stderr.closed:  # pylint: disable=using-constant-test
         logging.disable()
-    elif sys.stdout.closed:  # pylint: disable=using-constant-test
+    elif not sys.stdout or sys.stdout.closed:  # pylint: disable=using-constant-test
```

Comment:
Even though we no longer close standard stream descriptors in the daemon, I decided to leave this code to try to support when dvc is somehow run without the streams.

We could avoid this in logging, but IMO this is the best place to do this.